CODE_OF_CONDUCT.md

```diff
@@ -1,100 +1,103 @@
 
-Proposals and improvements must be completed for all weeks unless a week's content specifies otherwise in the accompanying assignment `README`. 
+### Improvements
 
-#### `A-` grades
+**Opportunities to implement improvements will begin during the `Bodega` assignment.**
 
-Citizens who do not meet the above requirements for two (2) weeks will be awarded a mark of `A-` if all other weeks contain a proposal and subsequent improvement(s).
+Much of the language below will not make complete sense until we're into the collaborative portion of our work. That will come a bit later.
 
-### The `B` tier
+Improvements are small additions to assignments that extend your service in `term-world`. They serve as a way to:
 
-(To qualify for this tier, all basic assignment objectives must be completely met.)
+* gain extra practice with course concepts
+* demonstrate an above-average level of effort
 
-Citizens who take the opportunity to both provide substantive improvements to assignment content qualify for this contract tier. In order to qualify for this tier, citizens must:
+Each "improvable" assignment will suggest `10` potential improvements to choose from. Selecting from this list, improvements must be unique (i.e. two students cannot complete the same improvement).
 
-* propose an assignment-related world improvment in each week's repository using the `Improvement Proposal` Issue template on GitHub which:
-  * completes all document prompts
-  * must be finished by the start of a lab session (Friday) of a given week
-  * must be recognized _in writing_ by the Mayor by the end of the lab session
-  * must either incorporate or provide rationale for _not_ incorporating suggested revisions
-    * this work is due by the assignment deadline (Sunday of each week)
+#### Improvement `Proposal`s
 
-These proposals may be revisions to existing content or creation of wholly new content. Proposals must be completed for all weeks unless a week's content specifies otherwise in the accompanying assignment `README`. 
+Improvements must go through a formal proposal process. To propose an assignment-related world improvment, citizens must use the `Improvement Proposal` template for a given GitHub repository's `Issues` which:
 
-#### `B+` grades
+  * responds to the templates questions/prompts
+  * is completed by the start of a lab session of a given week
+  * must be approved _in writing_ by the Mayor by the end of the lab session
+    * the Office of the Mayor will provide suggestions and approval via the `Issue` filed to propose the improvement
+  * must either incorporate suggested changes or provide rationale for _not_ incorporating them
 
-Citizens who both _propose and implement_ at least two (2) ideas will be awarded a mark of `B+` if all other weeks contain a proposal.
+#### Improvment `Implementation`
```

Comment:
Improvement

Reply:
I was thinking they'd have to make it up on the fly with the scenario that they're newscasters and one of them can't see the green screen behind them and the others try to give them hints but it's really hard because they also have to be funny and...

You're right. It needs an `e`.